freqtrade/optimize/backtesting.py

```diff
@@ -697,6 +798,8 @@ def backtest(self, processed: Dict,
             'config': self.strategy.config,
             'locks': PairLocks.get_all_locks(),
             'rejected_signals': self.rejected_trades,
+            # TODO: timedout_orders should be shown as part of results.
+            # 'timedout_orders': self.timedout_orders,
```

Comment:
@rokups asking your opinion again here ... you think a simple "cancelled" will suffice here - or should the value be split between entry and exit orders (that's the reason i've commented this out for now)

Reply:
I think the more statistics the better. Could be we have a rather reliable entry pricing and exits often timeout. If we lump those two together we would see just a number of timing out orders that looks about ok and would have no idea what actually is going on. We could then go on "improving" entry pricing only to make it worse (time out more) while we should have improved sell pricing instead..

Reply:
Err maybe if you are worried about statistics getting too crowded we could later on introduce a new command line switch to display advanced statistics. Then we can default to terse backtest result, but on user opting in we could display all kinds of stuff . cyberd3vil asked for more stats already.